File: web3rpc/rpc-specs/code-samples/python/eth/account/sign.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0x1cbd3b2770909d4e10f157cabc84c7264073c9ec"
message = "0xdeadbeaf"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.sign(address, message)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/account/sign.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const address = '0x487f2dfef230c2120b8cc55c5087b103146536ec'
    const message = '0xdeadbeaf'

    provider.eth.sign(address, message, {}, (err, data, response) => {
        console.log(data);
    });
}
)()